web/plugs/authorization.ex

```diff
@@ -75,12 +75,12 @@ defmodule Cog.Plug.Authorization do
   # in the future. For example, this would allow users to update
   # related data with paths such as `/v1/users/:user_id/profiles/:id`.
   defp self_updating?(opts, conn) do
-    if Keyword.get(opts, :allow_self_updates, false) == true do
-      conn.private.phoenix_action in [:update, :show] and
-      (conn.assigns.user.id == conn.params["id"] or
-       conn.params["id"] == "me")
-    else
-      false
+    case Keyword.get(opts, :allow_self_updates, []) do
+      [] -> false
+      actions ->
+        conn.private.phoenix_action in actions and
+        (conn.assigns.user.id == conn.params["id"] or
+         conn.params["id"] == "me")
```

Comment:
I realize this didn't change in this PR, but where do we ever have an `id` of `me`?